README.md

```diff
@@ -14,8 +14,35 @@ Documentation about Azure Spring Apps is available [here](https://learn.microsof
 
 ---
 
-## 🚀 How to get started
-
+## 🚀 How to generate a project
+### CLI
 1. Install the package with `npm i generator-jhipster-azure-spring-apps`
 1. Create and navigate to a directory
 1. Generate the application with `jhipster-azure-spring-apps createtodoapp`
+
+### JHipster Online
+Go to [Jhipster Online](https://start.jhipster.tech/generate-azure-application) and generate your application.
+
+## 🎉 How to deploy on Azure
+1. Install AZD, you can try one of the following options:
+ - **Windows package manager(winget)**: `winget install microsoft.azd`
+ - **Chocolatey**: `choco install azd`
+ - **Scrip**t: `powershell -ex AllSigned -c "Invoke-RestMethod 'https://aka.ms/install-azd.ps1' | Invoke-Expression"`
+1. Prepare environment
+</br>Log in Azure with OAuth2:
+</br> `azd auth login`
+</br>  Provision the template's infrastructure to Azure.
```

Comment:
why do we need to show 3 AZD CLI commands here instead of using `azd up` to cover them all? I suppose the reason that we use AZD is to help users to run fewer commands?

Reply:
does the user download the whole project and step into the parent foler of `infra` folder in the previous steops? We need to make sure the below steps of `azd up` is run in the correct position.

Reply:
In the quickstart, we break down the `azd up` into three steps, because we want to be consistent with the quickstart's structure. But I think in this doc, we can use `azd up` directly.